docs/contributing.rst

```diff
@@ -57,26 +57,6 @@ If testing for the same Airflow and Python version, next runs of the integration
 
     hatch run tests.py3.8-2.5:test-integration
 
-.. code-block:: bash
-
-    docker run --name postgres -p 5432:5432 -p 5433:5433 -e POSTGRES_PASSWORD=postgres postgres
-
-To run the integration tests for the first time, use:
-
-.. code-block:: bash
-
-    export AIRFLOW_HOME=`pwd`
-    export AIRFLOW_CONN_AIRFLOW_DB=postgres://postgres:postgres@0.0.0.0:5432/postgres
-    hatch run tests.py3.8-2.5:test-integration-setup
-    hatch run tests.py3.8-2.5:test-integration
-
-If testing for the same Airflow and Python version, next runs of the integration tests can be:
-
-.. code-block:: bash
-
-    hatch run tests.py3.8-2.5:test-integration
```

Comment:
@raffifu I just realised this is not necessarily a duplicate from the block below but an alternative.
People may opt to develop Cosmos without Docker Compose - since it is easier to troubleshoot and use debuggers..! We can improve the docs to make this clear, but I think we should still keep documented both approaches.

Reply:
@tatiana The block code is already present on the previous line, so i think it's a duplicate

https://github.com/astronomer/astronomer-cosmos/blob/dac117a36209729519add1ad99fe3418572c6aa7/docs/contributing.rst?plain=1#L56-L58

Anyway i have an idea to add instruction `astronomer-cosmos` without docker compose (directly on host machine). Is it possible to add the instruction on docs? but i need to try first to see if my approach is worked

Reply:
Absolutely, would love to see that, @raffifu !

Reply:
@tatiana Already pushed the change on the new commit